elementor/elementor
# Get data if it's empty

## PR Checklist



## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

*

## Description
https://github.com/elementor/elementor/issues/12928
I got one error because : The $datavariable is empty in  api.php line 158,this $data is received from elementor api endpoint and is saved to elementor_remote_info_api_data_ transient.We doesn’t delete or change in this transient in out projects.It only changes by it’s elementor plugin codes. elementor only check this endpoint data If the data is false and our data is empty now.
Because we got an error
with this changes everything will be ok
*

## Test instructions
Set elementor_remote_info_api_data_ELEMENTOR_VERSION transient to empty.
After that you can see this error when you open elementor editor : (pages => new page with elementor)
With merging this request you can see this error is removed.


*

## Quality assurance


Fixes #
